wpimath/src/main/native/include/frc/kinematics/SwerveModuleState.h

```diff
@@ -21,5 +23,24 @@ struct SwerveModuleState {
    * Angle of the module.
    */
   Rotation2d angle;
+
+  /**
+   * Optimize a desired state by deciding weather to reverse the direction the
+   * wheel spins in order to minimize travel time. This means that the furthest
+   * a wheel will ever rotate is 90 degrees, if used in conjunction with the
+   * PIDController class' continuous input functionality.
+   * 
+   * @param desiredState The desired state.
+   * @param currentAngle The current module angle.
+   */
+  static SwerveModuleState Optimize(const SwerveModuleState& desiredState,
+                                    const Rotation2d& currentAngle) {
+    auto delta = desiredState.angle - currentAngle;
+    if (units::math::abs(delta.Degrees()) > 90_deg) {
+      return {desiredState.speed * -1.0,
```

Comment:
Does the following work?
```suggestion
      return {-desiredState.speed,
```